Is reverseGeocode required?

According to README `reverseGeocode` is optional
https://github.com/maplibre/maplibre-gl-geocoder/blob/47bd8596eb29ee62eebdfcf35c7159f5accb9791/README.md?plain=1#L26-L27

But looks required in https://github.com/maplibre/maplibre-gl-geocoder/blob/47bd8596eb29ee62eebdfcf35c7159f5accb9791/lib/index.ts#L255

I got this error:
```
error TS2345: Argument of type '{ forwardGeocode: (config: GeocoderConfig) => Promise<{ features: { type: string; geometry: { type: string; coordinates: any[]; }; place_name: any; properties: any; text: any; place_type: string[]; center: any[]; }[]; }>; }' is not assignable to parameter of type 'MaplibreGeocoderApi'.
  Property 'reverseGeocode' is missing in type '{ forwardGeocode: (config: GeocoderConfig) => Promise<{ features: { type: string; geometry: { type: string; coordinates: any[]; }; place_name: any; properties: any; text: any; place_type: string[]; center: any[]; }[]; }>; }' but required in type 'MaplibreGeocoderApi'.

196    const newMaplibreGeocoder = new MaplibreGeocoder(nominatimGeocoder, {
                                                        ~~~~~~~~~~~~~~~~~

  node_modules/@maplibre/maplibre-gl-geocoder/dist/maplibre-gl-geocoder.d.ts:216:5
    216     reverseGeocode: (config: MaplibreGeocoderApiConfig) => Promise<{
            ~~~~~~~~~~~~~~
    'reverseGeocode' is declared here.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ### Features / Improvements 🚀
 
 - Migrate unit tests to Vitest https://github.com/maplibre/maplibre-gl-geocoder/pull/238
+- Make the `reverseGeocode` field in `MaplibreGeocoderAPI` optional https://github.com/maplibre/maplibre-gl-geocoder/pull/245
 
 ### Bug fixes 🐛
 

--- a/lib/index.ts
+++ b/lib/index.ts
@@ -252,7 +252,7 @@ export type MaplibreGeocoderApi = {
   /**
    * Reverse geocode function should return an object including a collection of {@link CarmenGeojsonFeature}.
    */
-  reverseGeocode: (config: MaplibreGeocoderApiConfig) => Promise<MaplibreGeocoderFeatureResults>;
+  reverseGeocode?: (config: MaplibreGeocoderApiConfig) => Promise<MaplibreGeocoderFeatureResults>;
   getSuggestions?: (config: MaplibreGeocoderApiConfig) => Promise<MaplibreGeocoderSuggestionResults>;
   searchByPlaceId?: (config: MaplibreGeocoderApiConfig) => Promise<MaplibreGeocoderPlaceResults>;
 };

